src/components/tipRecords/TipRecord.vue

```diff
@@ -110,23 +110,49 @@
             :src="tipPreviewImage"
             :onerror="`this.className+=' fail'`"
             :loading="`lazy`"
-            class="preview__image"
+            class="author_img preview__image"
           >
         </div>
       </div>
-      <div
-        v-else
-        class="tip__url"
-      >
-        <a
-          :href="tip.url"
-          :title="tip.url"
-          class="text-ellipsis"
-          target="_blank"
+      <div v-else>
+        <div v-if="tip.url">
```

Comment:
I don't understand, why should `template` be used here?

Reply:
I think we should prefer templates over divs if we don't need to have a DOM node inserted, but just to group several child nodes by the same condition. This should make refactoring easier because templates can't break layout or styles.

Reply:
for the v-if block?